src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+export default class App extends React.Component {
+    state ={
+        result :{},
+        loading: true
+    }
+    
+   async componentDidMount(){
+       const url="http://www.omdbapi.com/?t=the&apikey=d3a8db41"
+       const response= await fetch(url)
```

Comment:
Can we use Axios instead? This is a Mehrdad Personal preference :) and based on my experience it is easier to Unit test, but If you feel string about fetch I am fine with Fetch too. Just FYI, both are good but community like to use Axios more.